mtz-wizard-stepper.html

```diff
@@ -0,0 +1,43 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="mtz-wizard-stepper-behavior.html">
+
+<!--
+`mtz-wizard-stepper`
+A basic implementation of the `mtz.WizardStepperBehavior` for navigating an `mtz-wizard` element.
+
+@demo demo/index.html
+-->
+<dom-module id="mtz-wizard-stepper">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+    <template is="dom-repeat" items="[[__steps]]">
```

Comment:
For 2.x compliance don't we need to wrap <template is="dom-repeat"> ... within <dom-repeat> ...? I.e, <dom-repeat><template is="dom-repeat"> ... </></>?